Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -172,6 +172,11 @@ static RCPurchasesErrorCode RCPurchasesErrorCodeFromRCBackendErrorCode(RCBackend
         case RCBackendInvalidSubscriberAttributes:
         case RCBackendInvalidSubscriberAttributesBody:
             return RCInvalidSubscriberAttributesError;
+        case RCBackendPlayStoreInvalidPackageName:
```

Comment:
@aboedo @vegaro 

* could `RCBackendProductIdForGoogleReceiptNotProvided` be thrown from an ios app? wasn't sure on that one.
* i noticed that ios doesn't handle two backend error codes that android does:
BackendInternalServerError(7110) and BackendBadRequest(7226). 

i can add them with this PR, unless there was some reason for omitting them?

Reply:
I don't think `RCBackendProductIdForGoogleReceiptNotProvided` would ever be thrown but who knows. I think it is a good idea to add those `BackendInternalServerError(7110)` and `BackendBadRequest(7226)`, specially because if it ever gets received by one of the hybrid SDKs, iOS and Android would return a different error.